psyneulink/core/compositions/composition.py

```diff
@@ -10395,7 +10395,7 @@ def learn(
         from psyneulink.library.compositions import AutodiffComposition
         runner = CompositionRunner(self)
 
-        if (execution_mode in {pnlvm.ExecutionMode.PyTorch, pnlvm.ExecutionMode.LLVM}
+        if ((execution_mode is not pnlvm.ExecutionMode.Python)
```

Comment:
I think this will complain about using `is` instead of `==`.

Reply:
Comparing enum member not its value.